CONTRIBUTING.md

```diff
@@ -42,7 +42,8 @@ You should open an issue to discuss your pull request, unless it's a trivial cha
 
 1. Fork & Clone this repo (Make sure to disable associated GitHub Actions. In fork go to Settings > Actions > General > Disable actions > save)
 1. [`nvm install`](https://github.com/nvm-sh/nvm)
-1. [`nvm use`](https://github.com/nvm-sh/nvm)
+1. [`nvm use`](https://github.com/nvm-sh/nvm) (uses `.nvmrc` to use supported Node version)
+1. `yarn install`
```

Comment:
`yarn install` is run as part of `yarn setup`  
```suggestion
```

Reply:
Ah, so it is.  I might argue that `yarn setup` is a little idiosyncratic, but I'm not going to tilt at that windmill.  I'll correct the documentation accordingly.

Reply:
Corrected & PR amended